fiftyone/migrations/revisions/v1_4_0.py

```diff
@@ -0,0 +1,53 @@
+"""
```

Comment:
Please put a comment for why we are doing this. Either in module docstring or as a code comment